index.html

```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
            <img id="img" src="./features/images/Disabled.png">
       </section>
      </main>
+     <section>
+        <div id="content2"><p class="works">My Recent Works</p>
```

Comment:
- [ ] please, maybe you forgot. Like the previous reviewer noted, kindly make the `my recent works` title an `hx`